docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
-
-- Local: This profile is used when you are connected from the same network that the dappnode.
-- Remote: This profile is used when you are connected from a different network that the dappnode.
-  When you create the vpn profile there is a link that generate the profile for local or remote.
-
-## Can I recover my admin password?
-
-You can't recover your admin password, but you can access to your dappnode if you wrote down the recovery token, and then change the admin password.
-
-The first time you access your DAppnode,it appears the next image:
-
-<p align="center">
-    <img src="../../../img/sign_in_image.png"/>
-</p>
-
-Once you fill the fields and press the register button, DAppNode provides you with a recovery code. If you forget te DAppNode's Password you will be able to access using this token.
-
-<p align="center">
-    <img src="../../../img/recovery_token.png"/>
-</p>
-
-You should save your token in a secure place like a password manager, paper, etc. In this case, we would save this:
-
-```
-4LMB9w3l50Yljwr6bIgQ
-```
-
-## Can I recover my recovery token?
-
-Firstly, we have to go to the login page of our DAppNode:
-
-<p align="center">
+<details>
+  <summary>Help! I can't access my DAppNode!</summary>
+  <div>
+    <div>
+    If it is your first time trying to access your DAppNode, please try one of the following steps:
+    <br /><br />
+<ul>
+    <li>If you are connected to the same network as your DappNode via wired Ethernet, try to access the UI using the address <a href="http://dappnode.local">http://dappnode.local</a></li>
+    <li>If you've bought a DAppNode or installed it on a machine with integrated Wi-Fi, try to access the Wi-Fi hotspot 'DAppNodeWifi'.</li>
+</ul>
+
+If all of the above does not work, visit our <a href="https://discord.com/invite/dappnode">Discord</a> and ask for help in the #support channel.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Why can't I connect to my DAppNode via VPN?</summary>
+  <div>
+    <div>
+    If you have set up OpenVPN and you cant access your DAppNode, remember that you might need to set up port forwarding on your router. Sometimes the feature called 'UPnP' enables these ports on your router automatically but this sometimes does not work or is not available on your router.
+    <br /><br />
+
+To do this, you need to access your router configuration and "port-forward" the following ports: TCP Port 8092 and UDP Port 1194. 
+
+In case you have set up Wireguard, you might have to create two profiles:
+
+<ul>
+    <li>A Local profile: This profile is used when you are connecting from within the same network that the DAppNode resides in.</li>
+    <li> A Remote profile: This profile is used when you are connecting from a different network than the one where your DAppNode resides in.</li>
+</ul>
+  When you create the VPN profiles, there is a link that generates the profile for either local usage or remote usage.
```

Comment:
Not the case with wireguard, the preferred VPN

Reply:
no link generated that is.

Reply:
When checking a Wireguard profile, there is the option to `Show local credentials`. I will attach a screenshot here to make it easier to understand.

Reply:
most of these guides are about CLI setup

Reply:
Could you explain what you mean here? Basic troubleshooting guides should probably not be about CLI setup and since there's an option in the GUI, telling the users about it is a good way to go I'd wager. I've since added a screenshot showing the option. @alexpeterson91

Reply:
And reworded that sentence a bit

Reply:
Agreed but I'm going off of what's already on this page. There's multiple CLI arguments already here.

Reply:
And most of the troubleshooting issues where you can't access your machine do require direct console or privileged SSH access. Yeah GUI comes first when troubleshooting but I'd wager most access issues people have keep them from getting to the adminUI

Reply:
I tend to disagree that there are a lot of CLI steps in this document as of right now but I agree that we need to discuss CLI based recovery steps as well. I wouldn't wanna cram it into this PR though. I will see what I can do and open a seperate one.